website/docs/quick-start.md

```diff
@@ -46,16 +32,16 @@ The update all functionality allows you to address discrepancies that may arise
 
     If an instance doesn't exist or that instance doesn't support all the features copa needs the next will be attempted. Please see [custom buildkit addresses](custom-address.md) for more information.
 
-    After setting up the buildkit instance, run the following command to patch the image:
+2. After setting up the buildkit instance, run the following Copa command to patch the supplied image:
```

Comment:
let's keep the trivy sections here. so the flow would be

- set up buildkit
- trivy scan
- copa patch
  - copa patch all
  - copa patch trivy report
- trivy scan